packages/mui-lab/src/TreeView/TreeView.test.js

```diff
@@ -149,6 +149,26 @@ describe('<TreeView />', () => {
     expect(handleKeyDown.callCount).to.equal(3);
   });
 
+  it('should select node when a key is pressed ', () => {
+    const handleKeyDown = spy();
+
+    const { getByRole, getByTestId } = render(
+      <TreeView onKeyDown={handleKeyDown}>
+        <TreeItem nodeId="one" label="test" data-testid="one" />
+      </TreeView>,
+    );
+    act(() => {
+      getByRole('tree').focus();
+    });
+
+    expect(getByTestId('one')).not.to.have.attribute('aria-selected');
+
+    fireEvent.keyDown(getByRole('tree'), { key: 'ArrowDown' });
```

Comment:
```suggestion
```
It should be enough to just focus the element and press enter.

Reply:
Yes, you are right, I've checked it